<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WarGame</class>
 <widget class="QMainWindow" name="WarGame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1018</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WarGame</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>380</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="3">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>850</width>
        <height>100</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="1" colspan="2">
     <spacer name="verticalSpacer_4">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>180</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="1" rowspan="2" colspan="2">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="startButton">
        <property name="minimumSize">
         <size>
          <width>180</width>
          <height>83</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>180</width>
          <height>83</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/image/startPage/res/image/startPage/startButton.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>4</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="levelsButton">
        <property name="minimumSize">
         <size>
          <width>180</width>
          <height>83</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>180</width>
          <height>83</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/image/startPage/res/image/startPage/levelsButton.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>4</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="settingsButton">
        <property name="minimumSize">
         <size>
          <width>180</width>
          <height>83</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>180</width>
          <height>83</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/image/startPage/res/image/startPage/settingsButton.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_6">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>4</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="helpButton">
        <property name="minimumSize">
         <size>
          <width>180</width>
          <height>83</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>180</width>
          <height>83</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/image/startPage/res/image/startPage/helpButton.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>4</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="exitButton">
        <property name="minimumSize">
         <size>
          <width>180</width>
          <height>83</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>180</width>
          <height>83</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/image/startPage/res/image/startPage/exitButton.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="1">
     <spacer name="verticalSpacer_5">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>100</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1018</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>exitButton</sender>
   <signal>released()</signal>
   <receiver>WarGame</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>530</x>
     <y>616</y>
    </hint>
    <hint type="destinationlabel">
     <x>687</x>
     <y>656</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
